### DISCUSSION TITLE
Technosols

### DISCUSSION THREAD

**Discussion by StephanMantelISRIC** [iuss-wrb/wrb#8] [Reference Soil Groups]
This discussion is to document the RSG Technosols and document examples from sites and soils of Technosols with identified diagnostics and qualifiers

**Comment by Juilleret** [comment]
Dear all, 

This is my first time working with Technosols, and I'd like to understand how to annotate the horizons.

For example, here I have a Technosol with an asphalt layer that I've named Ru (for R, which resembles a rock, and "u" for urban), then a layer of fill made of fairly loose sand and slag from blast furnaces, which I've annotated 2Cu (2 because it's a new "anthropogenic sedimentation"), and then an earthy material similar to the Bg of the natural soil identified next to it, which appears to have been compacted and possibly reworked with a mechanical digger. Since no aggregates are visible anymore, I've annotated it 3Cg. Does this sequence Ru, 2Cu, 3Cg seem correct to you?

Thank you.

Jérôme Juilleret


<img width="1280" height="720" alt="cimetiere technosol" src="https://github.com/user-attachments/assets/925f324e-e873-457d-8668-4caa9e871394" />
